ui_framework/components/tabs/tab.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const KuiTab = ({ isSelected, onClick, children, className, ...rest }) => {
+  const classes = classNames('kuiTab', className, {
+    'kuiTab-isSelected': isSelected
+  });
+
+  return (
+    <button
+      className={classes}
+      onClick={onClick}
+      {...rest}
+    >
+      {children}
+    </button>
+  );
+};
+
+KuiTab.propTypes = {
+  isSelected: PropTypes.bool,
+  onClick: PropTypes.func,
+  children: PropTypes.node,
+  className: PropTypes.string
+};
```

Comment:
onClick should be required: a tab is to be selected and not only to represent something.
The default of isSelected should be false.
@weltenwort What should be the default of children? The string "tab" or "undefined" ?
className has no defaultProps in other components either: it's really optional.